src/terraform.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  name: Install Terraform, terraform-docs, and Ansible
+  become: yes
+  become_method: sudo
+  roles:
+    - role: terraform
+      vars:
+        # According to the RTA folks they are on 0.14.  This is the
+        # most recent 0.14 version.
+        terraform_version: 0.14.11
+    - role: assessment_tool
+      vars:
+        group: vnc
+        install_dir: /home/vnc/
+        owner: vnc
```

Comment:
Don't assume that VNC user's username is always going to be`vnc`, even though that is the [default in cisagov/ansible-role-vnc-server](https://github.com/cisagov/ansible-role-vnc-server/blob/9f7886e7067fddb069344d814e304ad06345f67f/defaults/main.yml#L8-L9).  You should grab the username from SSM the same way as in the [VNC playbook](https://github.com/cisagov/terraformer-packer/blob/fd7edf26bd44705b16b251c71583a4343ca4cd21/src/vnc.yml#L98).

Reply:
Great catch!  See commit cadd83b.